packages/client-core/src/admin/components/Project/ProjectFields.tsx

```diff
@@ -326,8 +327,11 @@ const ProjectFields = ({ inputProject, existingProject = false, changeDestinatio
   ])
 
   useEffect(() => {
-    if (projectUpdateStatus?.value?.triggerSetDestination?.length > 0) {
-      ProjectUpdateService.setDestinationURL(project, projectUpdateStatus.value.triggerSetDestination)
+    if (
+      projectUpdateStatus?.value?.triggerSetDestination?.length > 0 &&
+      projectUpdateStatus?.value?.destinationURL?.length === 0
```

Comment:
@barankyle Am I right in assuming that `destinationURL` will only be set through this useEffect when the project drawer is opened?

Reply:
I believe so, I'm pretty sure the drawer is not rendered at all when it's closed, and therefore the useEffect is not firing at that time.